app/models/supplejack_api/search.rb

```diff
@@ -43,6 +44,17 @@ def facet_list
       @facet_list
     end
 
+    def facet_pivot_list
+      return @facet_pivot_list if @facet_pivot_list
+
+      @facet_pivot_list =
```

Comment:
This could be better, but Sunspot doesn't provide a simple way to get the solr suffixed name for dynamic fields